RELEASE_NOTES.md

```diff
@@ -1,5 +1,521 @@
+
+This allows you replace forcing resources to run at compile time:
+
+```ruby
+  my_resource "foo" do
+    action :nothing
+  end.run_action(:run)
+```
+
+With the simpler compile_time property:
+
+```ruby
+  my_resource "foo" do
+    compile_time true
+  end
+```
+
+### build_essential
+
+The `build_essential` resource includes a new `:upgrade` action for macOS systems that allows you to install updates to the Xcode Command Line Tools available via Software Update.
+
+### cron
+
+The `cron` resource has been updated to use the same property validation for cron times that the `cron_d` resource uses. This improves failure messages when invalid inputs are set and also allows for `jan`-`dec` values to be used in the `month` property.
+
+### dnf_package
+
+The `dnf_package` resource, which provides `package` under the hood on any system shipping with DNF, has been greatly refactored to resolve multiple issues. The version behavior and overall resource capabilities now match that of the `yum_package` resource.
+
+- The `:lock` action now works on RHEL 8
+- Fixes to prevent attempting to install the same package during each Chef Infra Client run.
+- Resolved several idempotency issues.
+- Resolved an issue where installing a package with `options '--enablerepo=foo'` would fail.
+
+### git
+
+The `git` resource now fully supports why-run mode and no longer checks out the `deploy` branch by default as mentioned in the breaking changes section.
+
+### locale
+
+The locale resource now supports setting the system locale on Windows hosts.
+
+### msu_package resource improvements
+
+The `msu_package` resource has been improved to work better with Microsoft's cumulative update packages. Newer releases of these packages will not correctly install over previous versions. We've also extended the default timeout for installing MSU packages to 60 minutes. Thanks for reporting the timeout issue [@danielfloyd](https://github.com/danielfloyd)
+
+### package
+
+The package resource on macOS and Arch Linux systems now supports passing multiple packages into a single package resource via an array. This allows you to collapse multiple resources into a single resource for simpler cookbook authoring and it's also significantly faster as it requires fewer calls to the packaging systems. Thanks for the Arch Linux support [@ingobecker](https://github.com/ingobecker)
+
+Using multiple resources to install a package:
+
+```ruby
+package 'git'
+package 'curl'
+package 'packer'
+```
+
+or
+
+```ruby
+%w(git curl packer).each do |pkg|
+  package pkg
+end
+```
+
+can now be simplified to:
+
+```ruby
+package %w(git curl packer)
+```
+
+### service
+
+The `service` resource has been updated to support newer releases of `update-rc.d` so that it properly disables sys-v init services on Debian Linux distributions. Thanks [@robuye](https://github.com/robuye)
```

Comment:
```suggestion
The `service` resource has been updated to support newer releases of `update-rc.d` so that it properly disables sys-v init services on Debian Linux distributions. Thanks [@robuye](https://github.com/robuye)!
```